.gitignore

```diff
@@ -0,0 +1,9 @@
+/node_modules/
```

Comment:
This can be simplified:

`/node_modules/*`
`.DS_Store`

Also consider setting up a global gitignore file: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore